app/lib/modules/login/service/login_service.dart

```
@@ -66,16 +71,19 @@ final class LoginService {
 
   /// Authenticates the user with biometrics or device authentication options available on the device.
   /// Returns a `Future<bool>` which is `true` if successful, `false` otherwise.
+  ///
   /// [localizedReason] is the message displayed to the user during the authentication prompt.
+  ///
+  /// Might throw a `PlatformException` if there were technical problems.
   Future<bool> localAuthenticate(String localizedReason, [bool stickyAuth = false]) {
     try {
       return localAuthentication.authenticate(
         localizedReason: localizedReason,
         options: AuthenticationOptions(useErrorDialogs: false, stickyAuth: stickyAuth),
       );
     } catch (e, s) {
-      Log.e('$e', 'LoginStore', s);
-      return Future.value(false);
```

This might have been the reason that on some phones the authentication didn't work any more. We swallowed the exception, and returned false for authentication failed.